lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -19,22 +19,36 @@
                 '<span class="member-count"><%= membership_message %></span>' +
                 '<ul class="list-member-thumbs"></ul>'
             ),
+            profileImageTemplate: _.template(
```

Comment:
My feeling is that we shouldn't have HTML in our JavaScript because that's mixing the two worlds. If this is a common thumbnail representation then it should be an Underscore file that other views can use.